### DISCUSSION TITLE
Issue in Terraform main.tf

### DISCUSSION THREAD

**Discussion by Maheshdvp** [learndevopsonline/learndevopsonline#1498] [Q&A]
Hi team , I'm getteing result like this i wanna perform terraform comments could you please help me in that
workshop;3.90.163.14
![image](https://github.com/learndevopsonline/learndevopsonline/assets/116950128/b73e130c-421c-4771-83f9-f3cb038affe2)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/116950128/4b78f689-f470-4244-8897-7431d0cd93d7)

**Comment by sreedharm07** [comment]
In the values why you are giving yes ? as a value.
you are giving values as yes thats why its giving error

you are just giving apply thats why its asking for values. you should give -var-file=env-dev/main.tfvars in apply command  because values are in main.tfvars

**Reply by Maheshdvp** [reply]
hi @sreedharm07 it was throwingb error like this when i had given make dev 
![image](https://github.com/learndevopsonline/learndevopsonline/assets/116950128/61bffd73-f18b-4727-8c91-38c458ca386d)

**Reply by sreedharm07** [reply]
not rf.terraform   
give command properly 
rm - rf.terraform is wrong 
rm -rf  .terraform is correct
